test/de/schildbach/pte/live/VrsProviderLiveTest.java

```diff
@@ -121,14 +120,16 @@ public void nearbyStationsWithLimits() throws Exception {
         print(result);
 
         final NearbyLocationsResult result2 = queryNearbyLocations(EnumSet.of(LocationType.STATION),
-                Location.coord(50732100, 7096820), 0, 50);
+                Location.coord(50732100, 7096820), 0, 1);
         print(result2);
 
         final NearbyLocationsResult result3 = queryNearbyLocations(EnumSet.of(LocationType.STATION),
-                Location.coord(50732100, 7096820), 1000, 50);
+                Location.coord(50732100, 7096820), 100, 0);
         print(result3);
     }
 
+    @Ignore
```

Comment:
Es the error 500 happening because of the parameters of our query (both 0), or because the result set would be empty?

In the second case I think we should handle this in the provider. HTTP status codes is not something the apps want to deal with.

Reply:
The Java method arguments types, maxDistance and maxLocations are not passed to the API, but are used for internal filtering. (When I have more time, I might reverse engineer the VRS app to find out if the API supports these parameters.)

The resulting URL for the test case is http://android.vrsinfo.de/index.php?eID=tx_ekap_here&ta=vrs&lat=0.000001&lon=0.000000 (a location far outside the VRS area).

The result body is:
```
{"fehler":"Ung\u00fcltige Umkreis-Koordinate"}
```

But unfortunately, the status code is 500 so that de.schildbach.pte.util.HttpClient.getInputStream throws an InternalErrorException, which cannot be handled gracefully by the provider implementation.

Reply:
I just checked that if both coordinates are != 0, a valid result set is returned.

So if you replace
```
Location.coord(1, 0)
```
with
```
Location.coord(1, 1)
```

the API returns four results from Switzerland: http://android.vrsinfo.de/index.php?eID=tx_ekap_here&ta=vrs&lat=0.000001&lon=0.000001

So please feel free to replace the Location.coord arguments, re-activate the test case and assert 4 results.

Reply:
...and if we now set maxDistance to 1000 m instead of 0 (unlimited), the original purpose of the test case is restored:

```
    @Test
    public void nearbyLocationsEmpty() throws Exception {
        final NearbyLocationsResult result = queryNearbyLocations(EnumSet.allOf(LocationType.class),
                Location.coord(1, 1), 1000, 0);
        print(result);
        assertEquals(0, result.locations.size());
    }
```